SPECS/samba.spec

```diff
@@ -51,8 +78,11 @@
 %endif
 
 %global with_mitkrb5 1
+%global with_dc 1
 
+%if %{with xcpng}
```

Comment:
The original `%if 0%{?rhel}` was good enough. We don't need our own macro here, as long as we build on a derivative of RHEL (and the day we'd not build on such a derivative, we'd have to review all the packaging anyway).